fractal/client/client.py

```diff
@@ -113,7 +162,8 @@ async def project_list():
         for p in project_list:
             if p.read_only:
                 p_read_only = "✅"
-            p_read_only = "❌"
+            else:
+                p_read_only = "❌"
```

Comment:
what does this code do? not immediately clear to me

Reply:
It simply maps p.read_only (True/False) to a pretty string ("✅"/"❌"). I added a comment and better variable names.